src/api/app/services/request_diff_cacher.rb

```diff
@@ -0,0 +1,8 @@
+class RequestDiffCacher
```

Comment:
If you just want to have this code in another file, use a concern please.

Reply:
I have reverted the change